tests/kicker/test_basic_commands.py

```diff
@@ -0,0 +1,81 @@
+from parameterized import parameterized
+import unittest
+
+from utils.channel_access import ChannelAccess
+from utils.testing import parameterized_list
+from hamcrest import assert_that, is_, equal_to
+
+DEVICE_PREFIX = "KICKER_01"
+
+# VOLTAGE CALIBRATION CONSTANTS FOR TESTS
+DAQ_MAX_VOLTAGE = 10.0
+PSU_MAX_VOLTAGE = 45.0  # This is the default value set as a macro in IOC st.cmd
+VOLTAGE_CALIBRATION_RATIO = PSU_MAX_VOLTAGE / DAQ_MAX_VOLTAGE
+
+# CURRENT CALIBRATION CONSTANTS FOR TESTS
+DAQ_MAX_CURRENT = 10.0
+PSU_MAX_CURRENT = 15.0  # This is the default value set as a macro in IOC st.cmd
+CURRENT_CALIBRATION_RATIO = PSU_MAX_CURRENT / DAQ_MAX_CURRENT
+
+
+class BaseTests(unittest.TestCase):
+    record = None
+    calibration = None
+
+    def setUp(self):
+        self.ca = ChannelAccess(20, device_prefix=DEVICE_PREFIX)
+        self.ca.assert_that_pv_is("DISABLE", "COMMS ENABLED")
+        self._simulate_value(0)
+
+    def _simulate_value(self, value):
+        pv_root = "DAQ:{}".format(self.record)
+        self.ca.set_pv_value("{}:SIM".format(pv_root), value)
+        self.ca.assert_that_pv_is("{}:SIM".format(pv_root), value)
+
+        value_to_check = self.ca.get_pv_value("{}:_RAW".format(pv_root))[0]
+        assert_that(value_to_check, is_(equal_to(value)))
+
+    @parameterized.expand(
+        parameterized_list([4.68, 6, 10, 0, 4e-3])
+    )
+    def test_that_GIVEN_a_value_THEN_the_calibrated_value_is_read(self, _, value_to_set):
```

Comment:
The names of these tests are very vague, which I think is because of the multiple purposes each test needs to conform to as part of the test suite. As a result the meaning of each test is hidden and can make them hard to read, and it is more difficult to diagnose a failed test. Similar for other tests.